roles/splunk_common/tasks/get_facts_service_name.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Kinda leaning towards handling Linux vs. Windows // Splunk vs UF service naming in the `inventory/splunk*_defaults_*.yml` files since that matrix of statically-defined per-platform, per-product changes already exist there. 

Also it may be nicer to do it there so you don't spend Ansible-provisioning-time setting up variables. I might take a stab at that later though.